libs/designsystem/src/lib/components/item/item.component.integration.spec.ts

```diff
@@ -75,9 +75,43 @@ describe('ItemComponent in Kirby List', () => {
       expect(firstItem).toHaveComputedStyle({ 'padding-top': size('xxs') });
       expect(lastItem).toHaveComputedStyle({ 'padding-bottom': size('xxs') });
     });
+
+    describe('with hasItemSpacing set to true', () => {
+      it('should apply spacing to all but the last item', () => {
+        spectator.setInput('hasItemSpacing', true);
+        spectator.detectChanges();
+        const kirbyItemsInList = spectator.queryAll('kirby-list-item:not(:last-child)');
+
+        kirbyItemsInList.forEach((item) => {
+          expect(item).toHaveComputedStyle({ 'margin-bottom': size('s') });
+        });
+      });
+
+      it('should not apply spacing to the last item', () => {
+        spectator.setInput('hasItemSpacing', true);
+        spectator.detectChanges();
+        const kirbyItemsInList = spectator.queryAll('kirby-list-item:last-child');
+
+        kirbyItemsInList.forEach((item) => {

```

Comment:
For good measure, consider adding something like expect(kirbyItemsInList.length).not.toBe(0) to make sure something is actually being retrieved

Reply:
Noice, I opted for `kirbyItemsInList.not.toBeEmpty()` to align with the other tests in this file.